mbbsd/cache.c

```diff
@@ -233,6 +233,12 @@ postperm_msg(const char *bname)
     assert(0<=i-1 && i-1<MAX_BOARD);
     bp = getbcache(i);
 
+#ifdef POST_COMMENT_WITH_TAIWAN_PHONE_ONLY
+    if (bp->brdattr & BRD_TAIWAN_PHONE_ONLY)
+        if (!is_taiwan_ip())
```

Comment:
Um... you are calling the wrong function. Does this even compile?

Reply:
Sorry for that, but actually, nope.

There are no any document for compile with AOTP version, so I just modify the code from previous version without testing.

Reply:
Btw, I am going to change it to `is_sms_verified`, is it right?

Reply:
Yes. Keep names and functionality in line with current code.

Reply:
There is no mention of AOTP in this repo. It's actually worded as SMS here. There is a doc here: `docs/register.md`